src/group_theory/group_action/subgroup.lean

```diff
@@ -0,0 +1,107 @@
+/-
+Copyright (c) 2021 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+-/
+import group_theory.group_action.basic
+import data.fintype.basic
+/-!
+# Conjugation action on subgroups
+
+This file defines the conjugation action of a group on its subgroups.
+-/
+
+open mul_action
+
+variables {G : Type*} [group G]
+
+/-- The conjugation action of `G` on the subgroups of `G` -/
+instance subgroup.mul_action' : mul_action G (subgroup G) :=
```

Comment:
Why the `'`? Is there another `subgroup.mul_action`? If so, should this one be a global instance?

Reply:
The other one is presumably `(S : subgroup G) [mul_action G M] : mul_action S M` which is just a naming conflict, not a semantic one.